device-types/Supermicro/SSG-6028R-E1CR16T.yml

```diff
@@ -0,0 +1,45 @@
+---
+manufacturer: Supermicro
+model: SuperStorage 6028r-E1CR16T
+slug: supermicro-superstorage-6028r-e1cr16t
+part_number: SSG-6028R-E1CR16T
+u_height: 2
+is_full_depth: true
+weight: 15.4
+weight_unit: kg
+airflow: front-to-rear
+comments: '[Technical Specification](https://www.supermicro.com/en/products/system/2U/6028/SSG-6028R-E1CR16T.cfm)'
+console-ports:
+  - name: Serial
+    type: de-9
+power-ports:
+  - name: PSU-1
+    type: iec-60320-c14
+    maximum_draw: 1000
+  - name: PSU-2
+    type: iec-60320-c14
+    maximum_draw: 1000
```

Comment:
Are these PSU's modular? If so, they should be modelled under `module-bays` and the associated PSU modules should be added to the module-types folder if they do not already exist